src/EventHub/EventHub.Autorest/custom/Set-AzEventHubNamespace.ps1

```diff
@@ -0,0 +1,345 @@
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.String]
+        ${DisableLocalAuth},
+
+        [Parameter(HelpMessage = "EncryptionKeySource")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.String]
+        ${EncryptionKeySource},
+
+        [Parameter(HelpMessage = "EncryptionKeyVaultProperty.")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.String]
+        ${EncryptionKeyVaultProperty},
+
+        [Parameter(HelpMessage = "EncryptionRequireInfrastructureEncryption")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.String]
+        ${EncryptionConfig}, #EncryptionConfig
+
+        [Parameter(HelpMessage = "IdentityType of EventHub Namespace.")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.String]
+        ${IdentityType},
+
+        [Parameter(HelpMessage = "IdentityUserAssigneIdentity")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.Collections.Hashtable]
+        ${IdentityId},#IdentityId
+
+        [Parameter(HelpMessage = "IsAutoInflateEnabled")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.String]
+        ${IsAutoInflateEnabled},
+
+        [Parameter(HelpMessage = "KafkaEnabled")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.String]
+        ${KafkaEnabled},
+
+        [Parameter(HelpMessage = "Location of EventHub Namespace")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.String]
+        ${Location},
+
+        [Parameter(HelpMessage = "MaximumThroughputUnit of Eventhub Namespace.")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.Int64]
+        ${MaximumThroughputUnit},
+
+        [Parameter(HelpMessage = "MinimumTlsVersion of Eventhub Namespace.")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.String]
+        ${MinimumTlsVersion},
+
+        [Parameter(HelpMessage = "PrivateEndpointConnectionName.")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.String]
+        ${PrivateEndpointConnection},
+
+        [Parameter(HelpMessage = "PublicNetworkAccess.")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.String]
+        ${PublicNetworkAccess},
+
+        [Parameter(HelpMessage = "The SkuCapacity of EventHub Namespace.")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.Int64]
+        ${SkuCapacity},
+
+        [Parameter(HelpMessage = "The SkuName Of EventHub Namespace.")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.String]
+        ${SkuName},
+
+        [Parameter(HelpMessage = "SkuTier.")]
```

Comment:
Remove SkuTier. SkuTier and SkuName from Set-AzEventHubNamespace. They are not updatable. For New-AzEventHubNamespace, Hide SkuTier and provide only SkuName and equate in Tier to Name in the script.